docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  shop:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    command: >
+      sh -c "python manage.py migrate &&
+             python manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - ./:/app
```

Comment:
Do not mount the full project directory as a volume in the final version, it should be done for the development process